Add Route Choice (#402)
AequilibraE/qaequilibrae: docs/source/menus_in_detail/pathscomputation.rst

Now we can revisit the instructions for :ref:`siouxfalls-stacked-bandwidth`

Route choice
------------

With route choice sub-module it is possible to create choice sets through multiple algorithms

> and also assign trips to the network using the traditional path-size logit.

Reviewer: ```suggestion
as well as assign trips to the network using the traditional path-size logit.
```

> In the tab "Route choice model", we add the model configurations. It consists of

Reviewer: ```suggestion
In the tab "Route choice model", we add the model configuration. It consists of
```

> three different boxes. In the first box "*Choice set generation*", we input parameters to 

Reviewer: ```suggestion
three different boxes. In the first box "*Choice set generation*", we input parameters for 
```

> build the choice models. In the "*Route choice model"*, we add the parameters for the route 

Reviewer: ```suggestion
the choice set construction. In the "*Route choice model"*, we add the parameters for the route 
```

> choice model, such as the cost function and the path overlap factor (PSL/beta) value. Finally, in 

Reviewer: ```suggestion
choice model, such as the utility function and the path overlap parameter (PSL/beta) value. Finally, in 
```

"*Graph configuration*" we set up the graph used for computation.

.. image:: ../images/route_choice_1.png
    :width: 750
    :align: center
    :alt: Route Choice model tab

> In the tab "Critical analysis", we can select if we want to run one of select link or sub-area 

Reviewer: ```suggestion
In the tab "Critical analysis", we can select to run either a set of select link analysis or a sub-area 
```

> analysis. These analysis cannot be run at the same time in QAequilibraE. If you run sub-area 

Reviewer: ```suggestion
analysis. These analyses cannot be run at the same time in QAequilibraE. If you choose to run a sub-area 
```

analysis, all OD pairs with demand are considered for computation. To select only a few pairs
of interest, we encourage you to take a look at 
`Route choice with sub-area analysis <https://www.aequilibrae.com/develop/python/route_choice/_auto_examples/plot_subarea_analysis.html>`_ 

> at AequilibraE's python documentation and run this task outside QGIS.

Reviewer: ```suggestion
at AequilibraE's Python documentation and run this task outside QGIS.
```

.. image:: ../images/route_choice_2.png
    :width: 750
    :align: center
    :alt: Critical analysis tab

> Lastly, the tab "Workload" consists in running the workflows. The first box, "*Execute single*"

Reviewer: ```suggestion
Lastly, the tab "Workload" allows users to choose between three tasks. The first box, "*Execute single*"
```

> consists in computing the route choices between two different nodes and visualize it, while the

Reviewer: ```suggestion
consists of computing route choices between two different nodes and visualizing it, while the
```

> second box "*Matrix*" allows selecting the demand data used in the model, and which tasks we'll

Reviewer: ```suggestion
second box "*Matrix*" allows the selection of a travel demand matrix to be assigned using the route choice specified.
```

> execute: only build the choice sets or assign the demand for route choice.

Reviewer: ```suggestion
This option also allows the user to save choice sets to disk while performing route choice.
```
